src/org/usfirst/frc4904/robot/commands/HopperSetBallio.java

```diff
@@ -0,0 +1,10 @@
+package org.usfirst.frc4904.robot.commands;
+
+
+import org.usfirst.frc4904.robot.subsystems.Hopper;
+
+public class HopperSetBallio extends HopperSet {
```

Comment:
If we're gonna change it, I think it should be the other way around (change everything to `Ballio`) since it makes writing commands like `BallioOuttake` much easier.